### ISSUE TITLE
Feature: Add full graph traversal features to make color coding the graph easier

### ISSUE THREAD

**Issue by adaburrows** [lightningrodlabs/rea-playspace#85]
This is a big one, but it'll enable making some pretty awesome features in our app including tracking, tracing, and other kinds of graph traversals.

**Comment by fosterlynn** [comment]
>This is a big one

Yup, really big! :)  The most detailed and current doc is [here](https://www.valueflo.ws/algorithms/track/), need to do track though.  If you give some notice before you start working, we'll tackle that and try to save you some time.

**Comment by adaburrows** [comment]
Transcribing some handwritten notes:
* Grey is in process (or do-able)
* Black is done
* Red is blocked
* Should green or some other color indicate in process, while grey indicate doable or vice-versa?
